Cargo.toml

```diff
@@ -32,32 +32,202 @@ exclude = [
+clap_complete_fig = "4.5.0"
+colored = "2.0.0"
+comrak = "0.16"
+crossbeam-channel = "0.5"
+dap = "0.4.1-alpha1"
+dashmap = "5.4"
+derivative = "2.2.0"
+devault = "0.1"
+dialoguer = "0.11"
+dirs = "3.0.2"
+downcast-rs = "1.2.0"
+either = "1.9.0"
+ethabi = { package = "fuel-ethabi", version = "18.0.0" }
+etk-asm = { package = "fuel-etk-asm", version = "0.3.1-dev", features = [ "backtraces" ] }
+etk-ops = { package = "fuel-etk-ops", version = "0.3.1-dev" }
+extension-trait = "1.0.1"
+fd-lock = "4.0"
+filecheck = "0.5"
+fs_extra = "1.2"
+futures = { version = "0.3.24", default-features = false, features = ["std", "async-await"] }
+gag = "1.0"
+gimli = "0.28.1"
+git2 = { version = "0.19", features = ["vendored-libgit2", "vendored-openssl" ] }
+gix-url = { version = "0.27", features = ["serde"] }
+glob = "0.3.1"
+graph-cycles = "0.1.0"
+hashbrown = "0.13.1"
+hex = "0.4.3"
+horrorshow = "0.8.4"
+im = "15.0"
+in_definite = "1.0.0"
+include_dir = "0.7.3"
+indexmap = { version = "2.0.0", features = ["rayon"] }
+indoc = "2.0"
+insta = "1.39.0"
+ipfs-api-backend-hyper = { version = "0.6", features = ["with-builder"] }
+itertools = "0.10.3"
+lazy_static = "1.4"
+libp2p-identity = { version = "0.2.4", features = ["secp256k1", "peerid"] }
+libtest-mimic = "0.7.3"
+lsp-types = { version = "0.94", features = ["proposed"] }
+mdbook = { version = "0.4", default-features = false }
+miden = "0.3.0"
+miden-core = "0.3.0"
+minifier = "0.3.0"
+notify = "5.0.0"
+notify-debouncer-mini = "0.2.0"
+num-bigint = { version = "0.4.3", features = ["serde"] }
+num-traits = "0.2.16"
+object = { version = "0.32.2", features = ["write"] }
+once_cell = "1.18.0"
+opener = "0.5.0"
+parking_lot = "0.12.1"
+paste = "1.0.14"
+peg = "0.7"
+pest = "2.1.3"
+pest_derive = "2.1"
+petgraph = { version = "0.6", features = ["serde-1"] }
+phf = { version = "0.10.1", features = ["macros"] }
+pretty_assertions = "1.4.0"
+prettydiff = "0.6.4"
+proc-macro2 = "1.0.5"
+quote = "1.0.21"
+rand = "0.8"
+rayon = "1.7.0"
+rayon-cond = "0.3"
+regex = "1.10.2"
+reqwest = "0.12"
+revm = "2.3.1"
+ropey = "1.5"
+rpassword = "7.2"
+rustc-hash = "1.1.0"
+semver = { version = "1.0", features = ["serde"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_ignored = "0.1.9"
+serde_json = "1.0.91"
+serde_with = "3.3.0"
+serde_yaml = "0.9.27"
+serial_test = "3.0.0"
+sha2 = "0.10"
+sha3 = "0.10.8"
+shellfish = { version = "0.6.0", features = ["rustyline", "async", "tokio"] }
+slotmap = "1.0.7"
+smallvec = "1.7"
+strsim = "0.11.1"
+strum = { version = "0.24.1", features = ["derive"] }
+syn = { version = "1.0.99", features = ["derive", "extra-traits", "full" ] }
+tai64 = { version = "4.0", features = ["serde"] }
+taplo = "0.7"
+tar = "0.4.38"
+tempfile = "3"
+term-table = "1.3"
+termion = "2.0.1"
+textwrap = "0.16.0"
+thiserror = "1.0.30"
+tikv-jemallocator = "0.5"
+tokio = { version = "1.12", features = [ "fs", "io-std", "io-util", "macros", "net", "process", "rt", "rt-multi-thread", "sync", "time"] }
```

Comment:
I think we do not want to specify any features in this file.

If I understand correctly, this will include all of these features for every package that uses tokio since [inherited dependency features are additive](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#inheriting-a-dependency-from-a-workspace)

Reply:
agree we should enable features at the member level, it's pretty hard to reason about why all these features are enabled when defined in the workspace manifest.

Reply:
Sorry, I read that wrong. I was under the assumption the workspace lists all the features to allow, and each member choses which to enable from that list. Ok cool, removed.